packages/babel-helper-builder-react-jsx/README.md

```diff
@@ -10,6 +10,7 @@ type ElementState = {
   call?: Object; // optional call property that can be set to override the call expression returned
   pre?: Function; // function called with (state: ElementState) before building attribs
   post?: Function; // function called with (state: ElementState) after building attribs
+  compat?: boolean // true if React is in compat mode
```

Comment:
tagName: ? ^ too?

Reply:
What exactly is `compat` again? If it's just in an older version of React maybe we should just drop it?

Reply:
`compat` is probably a bad name but it's an all lowercase tag name (it's how you write DOM tags in ReactDOM)

The check itself is `!!tagName && /^[a-z]|-/.test(tagName)`

Reply:
Oh, wait, this is actually unrelated to `isCompatTag`; it's for the `react-jsx-compat` transform 🤔 

> Turn JSX into React Pre-0.12 function calls

Reply:
I'm 👍 for dropping the `compat` option. Re-visiting the README, I'm unsure of why `pre`, `post`, and `compat` are inside the `ElementState` type. These 3 are keys that should appear in options (which aren't typed). I'll reflect the changes in a commit.